shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/domain/usecase/GetAllFeedingsUseCase.kt

```diff
@@ -18,7 +18,7 @@ class GetAllFeedingsUseCase(
 ) {
 
     @OptIn(ExperimentalCoroutinesApi::class)
-    operator fun invoke(): Flow<List<Feeding>> {
+    operator fun invoke(shouldFetch: Boolean = false): Flow<List<Feeding>> {
```

Comment:
This parameter is not used inside the function

Reply:
Good catch, thank you! Fixed